### ISSUE TITLE
[Bug]Crash after disable/re-enabling Ublock addon

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8097]
## Prerequisites
Have the Ublock Origin add-on installed. 

## Steps to reproduce
1.  Navigate to any website. (check if Ublock is working)
2. Tap the 3dot button, and select "Add-ons manager" from the overlay
3. Tap the Ublock Origin Add-on and disable it

## Alternative Steps to reproduce
1. Navigate to any website.
2. Tap the 3dot button, and select "Add-ons manager" from the overlay
3. Tap the Ublock Origin Add-on and disable it
4. Resume the previously loaded website and by tapping the 3dot button check if the Add-on is disabled
5. Tap and open again the "Add-ons manager" 
6. Tap the Ublock Origin Add-on and re-enable it
7.  Open a new website and  and by tapping the 3dot button check if the Add-on is enabled

### Expected behavior
The Add-on is properly disabled/enabled, and the browser works properly.

### Actual behavior
After disable/re-enable the Add-on and visiting the newly loaded website Fenix crashes.

### Device information

* Android device:  LG Nexus 4 (Android 5.1.1
* Fenix version: Nightly #20360606 from 2/5

### Notes
►Video
![20200208_123415](https://user-images.githubusercontent.com/51314259/73826597-bf9d9280-4806-11ea-8acc-775f96c459b1.gif)

►Log
```kotlin
--------- beginning of crash
02-08 12:18:30.414 13702-13702/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.nightly, PID: 13702
    org.mozilla.geckoview.GeckoResult$UncaughtException: java.lang.IllegalStateException: Fragment InstalledAddonDetailsFragment{11cc1b2f} (07b0da48-f0dc-4bf1-9d58-ec6fdd81b7b5)} not attached to a context.
        at org.mozilla.geckoview.GeckoResult.dispatchLocked(GeckoResult.java:4)
        at org.mozilla.geckoview.GeckoResult.completeExceptionally(GeckoResult.java:4)
        at org.mozilla.geckoview.GeckoResult.lambda$thenInternal$2$GeckoResult(GeckoResult.java:10)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$gwCgOUK_EYQn2g6GolfZvo6A_WE.run(lambda)
        at org.mozilla.geckoview.GeckoResult.lambda$dispatchLocked$3(GeckoResult.java:2)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$PehjG2jgyDa_p37vZrKrSi2I94s.run(lambda)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5254)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
     Caused by: java.lang.IllegalStateException: Fragment InstalledAddonDetailsFragment{11cc1b2f} (07b0da48-f0dc-4bf1-9d58-ec6fdd81b7b5)} not attached to a context.
        at androidx.fragment.app.Fragment.requireContext(Fragment.java:2)
        at androidx.fragment.app.Fragment.getResources(Fragment.java:1)
        at androidx.fragment.app.Fragment.getString(Fragment.java:2)
        at -$$LambdaGroup$ks$3dk2TV2PDS17-TeNsA6Slr1xVgA.invoke(com.android.tools.r8.jetbrains.kotlin-style lambda group:14)
        at -$$LambdaGroup$ks$w64fKk4Jd8_-8DZtFDeHAmr--qw.invoke(com.android.tools.r8.jetbrains.kotlin-style lambda group:12)
        at -$$LambdaGroup$js$4QK-OblCgHvu_T5-JweEs6mE9DY.onValue(java-style lambda group:25)
        at org.mozilla.geckoview.GeckoResult.lambda$thenInternal$2$GeckoResult(GeckoResult.java:2)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$gwCgOUK_EYQn2g6GolfZvo6A_WE.run(lambda) 
        at org.mozilla.geckoview.GeckoResult.lambda$dispatchLocked$3(GeckoResult.java:2) 
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$PehjG2jgyDa_p37vZrKrSi2I94s.run(lambda) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:135) 
        at android.app.ActivityThread.main(ActivityThread.java:5254) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:372) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698) 
02-08 12:18:30.427 560-14890/? W/DropBoxManagerService: Dropping: data_app_crash (2142 > 0 bytes)
02-08 12:18:30.480 560-12422/? I/OpenGLRenderer: Initialized EGL, version 1.4
02-08 12:18:30.509 12597-12761/? I/TBIRD: ForegroundAppMonitor - --> pollUsageEventsAndStats()
02-08 12:18:30.516 12597-12761/? I/TBIRD: ForegroundAppMonitor - <-- pollUsageEventsAndStats(7 ms)
02-08 12:18:30.666 560-1120/? W/ActivityManager: getRunningAppProcesses: caller 10066 does not hold REAL_GET_TASKS; limiting output
02-08 12:18:30.907 560-582/? W/ActivityManager: Activity pause timeout for ActivityRecord{2217f4a7 u0 org.mozilla.fenix.nightly/.App t33 f}
02-08 12:18:30.991 560-582/? I/ActivityManager: Start proc 14900:org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashReporter/u0a734 for activity org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity
```

**Comment by ekager** [comment]
Few hundred events in Sentry: https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/7209207/?query=is:unresolved

**Comment by Amejia481** [comment]
@AndiAJ this should be fixed in master now, could you please take a look?

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20370604 from 2/6`** using a **LG Nexus 4 (Android 5.1.1)**

After approx. 20 disable/enable action no crash encountered.
